Core/Aspects/Logging/LogAspect.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Castle.DynamicProxy;
+using WebNetSample.Core.CrossCuttingConcerns.Logging;
+using WebNetSample.Core.CrossCuttingConcerns.Logging.Log4Net;
+using WebNetSample.Core.Utilities.Interceptors;
+
+namespace Core.Aspects.Logging
+{
+    public class LogAspect : MethodInterception
```

Comment:
Looks like you have quite in common items with `ExceptionLogAspect`. We definitely need inheritance of reusable items here. I advise you to create base abstract class (you may either call it `BaseLogAspect`) that will inherit `MethodInterception` and contain reusable items, like base constructor reusable logic, `GetLogDetail` methods. Each of inherited items will implement/override their own unique items